oracle/src/tx/test2.js

```diff
@@ -0,0 +1,20 @@
+const Web3 = require('web3')
```

Comment:
Is it required to have a module required for testing in `oracle/src/tx`?

Reply:
I have committed this file by mistake. It is not needed.